app/helpers/application_helper.rb

```diff
@@ -77,4 +77,16 @@ def time_period_class(item)
     return 'currently-valid' if item.currently_valid?
     return 'expired' if item.expired?
   end
+
```

Comment:
Yes, they probably should.

Problem is that sometimes you'd rather the full button vs the small button.